Dockerfile-alpine.template

```diff
@@ -0,0 +1,111 @@
+FROM php:7.1-%%VARIANT%%
+
+# entrypoint.sh dependencies
+RUN apk add --no-cache \
+    bash \
+    coreutils \
+    rsync
+
```

Comment:
I think you should merge the top 3 RUN commands into a single one to keep it a similar as possible to the debian one.

Reply:
This way is better for build caching. See also other images in the docker-library.
I can adjust the Debian images if required.